dev/create-release/release-build.sh

```diff
@@ -288,7 +290,9 @@ if [[ "$1" == "package" ]]; then
   declare -A BINARY_PKGS_EXTRA
   BINARY_PKGS_EXTRA["hadoop2.7"]="withpip"
   if ! is_dry_run; then
-    BINARY_PKGS_EXTRA["hadoop2.6"]="withr"
+    if [[ $SPARK_VERSION < "3.0." ]]; then
+      BINARY_PKGS_EXTRA["hadoop2.6"]="withr"
```

Comment:
Hmmm, now you're not building the R packaging on 3.0.

The old build was admittedly a bit odd. It only supported one "extra" arg so it build 2.6+R and 2.7+PIP. It seems we need to change that now...

Reply:
I see. Let me try to rewrite the script to support two args and attach these both to 2.7